skin.min.css

```diff
@@ -0,0 +1,35 @@
+.highlight {
+    background-color: #ddd;
+}
+
+button.btn {
+  border-radius: 10px;
+  border-style: solid;
+  border-width: 20px;
+  font-weight: normal;
+  line-height: 1.15;
+  text-align: center;
+  vertical-align: middle;
+}
+btn{ 
+   &--primary {

```

Comment:
I'm not sure why, but it looks like your compilation isn't working correctly. This `&` symbol shouldn't be here because it's not valid CSS syntax - it should be removed by the Sass compiler.

My only guess would be that because you've got a file called `skin.css` already it's somehow interfering. If you rename `skin.css` to something else (`test.css`), then make a change to `skin.scss`, that might fix it.

Reply:
Hi thanks for comments.
I did change the name of skin.css to test.css. but still does not work.

Reply:
Hmm, then I'm afraid I don't know what the problem is then. I can take a look on Sunday